docs/rules/func-call-spacing.md

```diff
@@ -0,0 +1,77 @@
+# require or disallow spacing between `function` identifiers and their invocations (func-call-spacing)
+
+(fixable) The `--fix` option on the [command line](../user-guide/command-line-interface#fix) automatically fixes problems reported by this rule.
+
+When calling a function, developers may insert optional whitespace between the function's name and the parentheses that invoke it. The following pairs of function calls are equivalent:
+
+```js
+alert('Hello');
+alert ('Hello');
+
+console.log(42);
+console.log (42);
+
+new Date();
+new Date ();
+```
+
+## Rule Details
+
+This rule requires or disallows spaces between the function name and the opening parenthesis that calls it.
+
+## options
+
+This rule has one string option:
+
+- `"never"` (default) disallows space between the function name and the opening parenthesis
+- `"always"` requires space between the function name and the opening parenthesis
+
+### never
+
+Examples of **incorrect** code for this rule with the default `"never"` option:
+
+```js
+/*eslint func-call-spacing: ["error", "never"]*/
+
+fn ();
+
+fn
+();
+```
+
+Examples of **correct** code for this rule with the default `"never"` option:
+
+```js
+/*eslint func-call-spacing: ["error", "never"]*/
+
+fn();
+```
+
+### never
+
+Examples of **incorrect** code for this rule with the `"always"` option:
+
+```js
+/*eslint func-call-spacing: ["error", "always"]*/
+
+fn();
+
+fn
+();
```

Comment:
Based on discussion, it seems that this example should be removed, since "always" should only allow non-newline spaces?